tests/runcards/protocols.yml

```
@@ -432,6 +432,13 @@ actions:
       beta_param: null
       nshots: 10
 
+  - id: allXY
```

```suggestion
  - id: allXY unrolling
```
I would suggest this change just to have a better understanding when running tests